src/tasks/daprCommandTaskProvider.ts

```diff
@@ -134,6 +134,8 @@ function createCommandLineBuilder(daprPathProvider: () => string, daprDefinition
             .withNamedArg('--metrics-port', daprDefinition.metricsPort)
             .withNamedArg('--placement-host-address', daprDefinition.placementHostAddress)
             .withNamedArg('--profile-port', daprDefinition.profilePort)
+            .withNamedArg('--resources-path', daprDefinition.resourcesPath)
+            .withArrayArgs('--resources-path', daprDefinition.resourcesPaths)
```

Comment:
Does this need to be like this:
```suggestion
            .withArrayArgs('--resources-paths', daprDefinition.resourcesPaths)
```
I'm guessing no but I want to double check there wasn't a copy-paste error here.

Reply:
Correct (i.e. no).  It's copied from the `daprd` task.  The first uses the now-deprecated `resourcesPath` string property (to specify only a single value) while the second uses the new `resourcesPaths` string array property to allow more than one specified path--the CLI allows it to be specified multiple times.